modules/juce_audio_devices/native/juce_ios_Audio.cpp

```diff
@@ -573,7 +573,7 @@ struct iOSAudioIODevice::Pimpl      : public AudioPlayHead,
         return session.inputGain == gain;
     }
     
-    void selectBackMic ()
+    void selectBackOrFrontMic (bool backMic)

```

Comment:
Could it be more readable to be using an enum here? This might also make it more flexible when adding other mic options? 

e.g.
```cpp
selectMicPosition(MicPosition::Back);
```